pages/learn/develop/blocks.md

```diff
@@ -111,14 +111,61 @@ By default, the block is set to `track latest`, which means that new releases ar
 
 ## Making your block public
 
-When a block is created on balenaCloud it will not automatically be publicly visible on balenaHub. When a block is made public it is listed in the [Blocks section][blockhub] of balenaHub, but is not joinable like fleets or forkable like projects. Instead the card for the block allows users to find the source code for the block, and learn how to deploy and use it in their own fleet.
+When a block is created on balenaCloud it will not automatically be publicly visible on balenaHub. When a block is made public it is listed in the [Blocks section][blockhub] of balenaHub, but is not joinable like fleets or forkable like projects. Instead the card for the block allows users to find the image URL for the block, and learn how to deploy and use it in their own fleet.
 
 When blocks are private, you can iterate, test and build more functionality before making it available to the world. When you are ready to release, head to the `Settings` tab in the sidebar and add your block's GitHub repository to the `Repository URL`  section. Next, toggle the `Block visibility` button to `on` for your block to be visible on balenaHub. You’re free to toggle the visibility on and off as necessary at any time.
 
 <img src="/img/blocks/settings.png" width="100%">
 
 This would lead to your block being available on [balenaHub][blockhub] for folks to use and build upon as soon as your submission has been marked as public. BalenaHub is the ever-growing library of blocks, projects and fleets contributed by the community.
 
+## Using your block in other projects
+
+The URL to pull your block image is available in balenaHub under the `Image Url` field.
```

Comment:
The field is called `Image Reference`

Reply:
Updated, but I'll need some new screenshots to go with it.

Reply:
@klutchell Changed Hub UI label and made screenshot [here](https://www.flowdock.com/app/rulemotion/balenahub/threads/5QSBmgacHBJ1YAry7Mh66XUt5vs)